api/document/js/main.ts

```diff
@@ -0,0 +1,83 @@
+import {Schema, Node} from "prosemirror-model";
+import {EditorView} from "prosemirror-view";
+import {EditorState} from "prosemirror-state";
+import {menuBar, undoItem, redoItem, MenuItem} from "prosemirror-menu";
+import {history, undo, redo} from "prosemirror-history";
+import {keymap} from "prosemirror-keymap";
+import {baseKeymap} from "prosemirror-commands";
+
+declare function require(path: string): null;
+
+require("prosemirror-view/style/prosemirror.css");
+require("prosemirror-menu/style/menu.css");
```

Comment:
You probably tried this, but would
```js
import "prosemirror-view/style/prosemirror.css";
```
work? That's valid ES6, but maybe not TypeScript.

Reply:
Yeah, that's what I originally tried, but for some reason typescript/webpack dies on it, claiming that it can't find that module.

I suspect it is because `ts-loader` (the typescript plugin for webpack) probably loads `import` statements on its own, without going through webpack, and it doesn't know what to do with non-standard kinds of imports like CSS, so using `require` likely bypasses typescript and goes straight to webpack, which deals with it. That's just my guess though.

That said, I should at least add a comment explaining this, so I'll do that!